packages/accordion/README.md

```diff
@@ -146,202 +169,191 @@ import { Accordion, AccordionItem } from '@spectrum-web-components/accordion';
-        <div>Item 1</div>
-    </sp-accordion-item>
-    <sp-accordion-item disabled label="Heading 2">
-        <div>Item 2</div>
+<sp-accordion allow-multiple>
+    <sp-accordion-item label="Kermit">
+        <div>Kermit is a frog.</div>
     </sp-accordion-item>
-    <sp-accordion-item label="Heading 3">
-        <div>Item 3</div>
+    <sp-accordion-item label="Fozzie">
+        <div>Fozzie is a bear.</div>
     </sp-accordion-item>
-    <sp-accordion-item label="Heading 4">
-        <div>Item 4</div>
+    <sp-accordion-item label="Miss Piggy">
+        <div>Miss Piggy is a pig.</div>
     </sp-accordion-item>
 </sp-accordion>
 ```
 
-</sp-tab-panel>
-</sp-tabs>
-
-### Spacious
+#### Disabled
 
-<sp-tabs selected="m" auto label="Size Attribute Options">
-<sp-tab value="s">Small</sp-tab>
-<sp-tab-panel value="s">
+Individual accordion items can be disabled using the `disabled` attribute. Disabled items cannot be expanded or collapsed.
 
 ```html
-<sp-accordion density="spacious" size="s">
-    <sp-accordion-item label="Heading 1">
-        <div>Item 1</div>
-    </sp-accordion-item>
-    <sp-accordion-item disabled label="Heading 2">
-        <div>Item 2</div>
+<sp-accordion>
+    <sp-accordion-item label="Apples">
+        <div>
+            We have some of the most popular varieties include Red Delicious,
+            Gala, Granny Smith, Honeycrisp, and Fuji.
+        </div>
     </sp-accordion-item>
-    <sp-accordion-item label="Heading 3">
-        <div>Item 3</div>
+    <sp-accordion-item disabled label="Bananas">
+        <div>We have the Gros Michel.</div>
     </sp-accordion-item>
-    <sp-accordion-item label="Heading 4">
-        <div>Item 4</div>
+    <sp-accordion-item label="Oranges">
+        <div>We have Mandarins, Seville Oranges, and Clementines.</div>
     </sp-accordion-item>
 </sp-accordion>
 ```
 
-</sp-tab-panel>
-<sp-tab value="m">Medium</sp-tab>
-<sp-tab-panel value="m">
+### Accessibility
 
-```html
-<sp-accordion density="spacious" size="m">
-    <sp-accordion-item label="Heading 1">
-        <div>Item 1</div>
-    </sp-accordion-item>
-    <sp-accordion-item disabled label="Heading 2">
-        <div>Item 2</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 3">
-        <div>Item 3</div>
-    </sp-accordion-item>
-    <sp-accordion-item label="Heading 4">
-        <div>Item 4</div>
-    </sp-accordion-item>
-</sp-accordion>
-```
+The accordion component provides proper ARIA attributes and keyboard navigation:
 
-</sp-tab-panel>
-<sp-tab value="l">Large</sp-tab>
-<sp-tab-panel value="l">
+- Each accordion item header has `aria-expanded` to indicate its current state
+- The header button has `aria-controls` pointing to the content region
+- The content region has `role="region"` and `aria-labelledby` pointing to the header
+- Disabled items have `aria-disabled="true"` applied
+- The accordion supports keyboard navigation with arrow keys and <kbd>Enter</kbd>/<kbd>Space</kbd> for activation
```

Comment:
Can we also mention if arrow keys are supported or not (many accordions support up/down navigation).